### PR TITLE
Add Realm Browser cask

### LINKED ISSUE #137: Caskroom Support
Can you support installation of Realm Browser using [caskroom](https://caskroom.github.io)?

### PR DESCRIPTION
**Pull request by chrisamanse**
Add [Realm Browser](https://github.com/realm/realm-browser-osx) cask.

This will also resolve realm/realm-browser#137.

### CODE REVIEW

File: Casks/realm-browser.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'realm-browser' do
+  version '0.98.2'
+  sha256 :no_check
```
Comment (victorpopkov) [inline Casks/realm-browser.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file. For the v0.98.2 it's:
```ruby
  sha256 '529d31d375a0a09b77d6c80531e3add99473e2b33ec457587698e148cee86abe'
```

File: Casks/realm-browser.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  url "https://github.com/realm/realm-browser-osx/releases/download/#{version}/RealmBrowser.app.zip"
```
Comment (victorpopkov) [inline Casks/realm-browser.rb]: An [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md) also should be specified with appropriate `checkpoint`:
```ruby
  url "https://github.com/realm/realm-browser-osx/releases/download/#{version}/RealmBrowser.app.zip"
  appcast 'https://github.com/realm/realm-browser-osx/releases.atom',
          checkpoint: 'a82f1e55a88ba8c3502d57e3c5ff18a77557fbd4770734de07652c119d38e30d'
```

File: Casks/realm-browser.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+cask 'realm-browser' do
+  version '0.98.2'
+  sha256 '529d31d375a0a09b77d6c80531e3add99473e2b33ec457587698e148cee86abe'
+
+  url "https://github.com/realm/realm-browser-osx/releases/download/#{version}/RealmBrowser.app.zip"
+  appcast 'https://github.com/realm/realm-browser-osx/releases.atom'
```
Comment (victorpopkov) [inline Casks/realm-browser.rb]: Here should be a comma at the end:
```ruby
  appcast 'https://github.com/realm/realm-browser-osx/releases.atom',
```

**Comment by chrisamanse** [comment]
Squashed into a single commit. Thanks.

**Comment by victorpopkov** [comment]
Thank you for the contribution. It was merged directly as c023d767b3c16462f1c90148cf35ff456faacd33 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**